### ISSUE TITLE
Audio and video of this lesson and preceding ones could be better

### ISSUE THREAD

**Issue by brigj1** [learn-co-curriculum/phase-3-lecture-videos-sql-update-and-delete#2]
### Canvas Link

https://learning.flatironschool.com/courses/5187/pages/video-sql-update-and-delete?module_item_id=398107

### Concern

I gave a "thumbs down" signal for this lesson. Content is fine, but the audio and visual quality make it hard to follow.

### Suggested Changes

Re-record? Maybe use a different SQL interface where font shows larger?

**Comment by lizbur10** [comment]
Hi @brigj1 - thanks for providing feedback on the videos. We are aware that there are problems with some of them and will re-record them as soon as we can.
